styles/Editor.css

```diff
@@ -954,6 +954,19 @@
 .ProseMirror .table-footer .footnote-element {
   counter-reset: table_footnote;
 }
+.table-footer .delete-table-footnotes {
+  position: absolute;
+  right: -20px;
+  top: -13px;
```

Comment:
Indeed, this is necessary to correctly position the icon since it has an absolute position. Additionally, assigning negative values to the absolute element is not a problem at all.